File: server-mocks/src/main/java/social/api/test/server/MocksRunner.java
package social.api.test.server;

import java.io.IOException;

/**
 * Entry point runs all mocking implementations.
 */
public class MocksRunner {
    private static final String DEFAULT_BASE_PATH = "http://127.0.0.1:8080";

    public static void main(String[] args) throws IOException {
        MocksServer server = new MocksServer(DEFAULT_BASE_PATH).start();
        System.in.read();
        server.shutdown();
    }
}

File: server-mocks/src/main/kotlin/social/api/test/server/MocksRunner.kt
package social.api.test.server

import java.io.IOException

val DEFAULT_BASE_PATH = "http://127.0.0.1:8080"

@Throws(IOException::class)
fun main(args: Array<String>) {
    val server = MocksServer(DEFAULT_BASE_PATH).start()
    System.`in`.read()
    server.shutdown()
}
